server/routes/bookedtoday/unexpectedArrivals/addPersonalDetailsController.ts

```diff
@@ -0,0 +1,35 @@
+import type { RequestHandler, Response, Request } from 'express'
+import { convertToTitleCase } from '../../../utils/utils'
+import { State } from '../arrivals/state'
+
+export default class AddPersonalDetailsController {
+  public view(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      res.render('pages/unexpectedArrivals/addPersonalDetails.njk', {
+        errors: req.flash('errors'),
+        data: req.flash('input')[0],
+      })
+    }
+  }
+
+  public submit(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      if (req.errors) {
+        req.flash('input', req.body)
+        return res.redirect('/manually-confirm-arrival/add-personal-details')
+      }
+
+      const { firstName, lastName, year, month, day, sex } = req.body
+      const dateOfBirth = `${year.padStart(4, '0')}-${month.padStart(2, '0')}-${day.padStart(2, '0')}`
```

Comment:
If this goes in after [this](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-ui/pull/201/files) , there's a new createDate function to build up dates as we were doing it all over